instrumentation/startup/src/main/java/io/opentelemetry/android/instrumentation/startup/InitializationEvents.java

```diff
@@ -5,14 +5,14 @@
 
 package io.opentelemetry.android.instrumentation.startup;
 
-import io.opentelemetry.android.config.OtelRumConfig;
 import io.opentelemetry.sdk.trace.export.SpanExporter;
+import java.util.Map;
 
 public interface InitializationEvents {
 
     void sdkInitializationStarted();
 
-    void recordConfiguration(OtelRumConfig config);
+    void recordConfiguration(Map<String, String> config);
```

Comment:
If I understand correctly, this is needed to avoid a circular dependency with the `android-agent` module, right?

Reply:
Yeah exactly.